### ISSUE TITLE
Complex Diagrams Are Unreadable in High Contrast or Dark Mode

### ISSUE THREAD

**Issue by kzjackson1** [knative/docs#1711]
## Expected Behavior
Ability to read diagrams in dark mode or high contrast mode.

## Actual Behavior
Because most of the images are .svg files with transparent backgrounds, black text or symbols are unreadable. 

## Steps to Reproduce the Problem

1. Go to the Knative Eventing page (or any page with diagrams) with any accessibility extension that allows for high contrast mode. (Alternatively, use an extension that triggers dark mode on your browser.)
1. Try to read the text/decipher arrow locations/etc.
1. Fail (or at least struggle, depending on the extension/software).

## Additional Info

High contrast mode is used for some people with visual impairments, and dark mode is used by some people with sensory issues, so this is an accessibility issue more than a vanity issue.

I've tested this on:
- Windows 10 High Contrast Mode
- Dark Theme for Chrome
- High Contrast (by Google)

With the same effect.

## Proposal(s)
- Change the background color of svg files from transparent and/or change the text color to something other than black.
- Make sure that the contents of an image are explained in the documentation.
- Making sure alt text clearly explains the image could mitigate the issue.

**Install information:**
 - Platform (GKE, IKS, AKS, etc.): N/A
 - Knative Version: N/A
